watertap/unit_models/uv_aop.py

```diff
@@ -456,13 +526,75 @@ def eq_electricity_demand_phase_comp(b, t, p, j):
                 / b.lamp_efficiency
             )
 
-        # TODO: add minimum electricity demand for multiple solutes
-        # @self.Constraint(
-        #     self.flowsheet().config.time,
-        #     doc="Constraints for minimum electricity demand of the UV reactor.",
-        # )
-        # def eq_minimum_electricity_demand(b, t):
-        #     return b.electricity_demand_minimum[t] == smooth_max(b.electricity_demand_phase_comp[t, p, j])
+        @self.Constraint(
+            self.flowsheet().config.time,
+            self.config.property_package.phase_list,
+            self.config.property_package.solute_set,
+            doc="Constraints for calculating maximum electricity demand for different phases.",
+        )
+        def eq_max_phase_electricity_demand(b, t, p, j):
+            if p == b.config.property_package.phase_list.first():
+                return (
+                    b.max_phase_electricity_demand[t, p, j]
+                    == b.electricity_demand_phase_comp[t, p, j]
+                )
+            else:
+                return b.max_phase_electricity_demand[t, p, j] == (
+                    smooth_max(
+                        b.max_phase_electricity_demand[
+                            t, b.config.property_package.phase_list.prev(p), j
+                        ],
+                        b.electricity_demand_phase_comp[t, p, j],
+                        b.eps_electricity,
```

Comment:
I suppose we are missing a test to check that the solution works as intended with more than one phase. Nevertheless, I think this was already an additional capability that you didn't need to add, so maybe we can just make an issue to keep track of any UV-related things we want to revisit in future PRs for later improvement. In this case, we can note that a test should be added to check electricity demand for multi-component and multi-phase example.

Reply:
I don't know what kind of waterborne pathogens is solid or gas phase, maybe we could add a dummy solute for verifying the test? e.g. solid `DCE`

Reply:
We could try something like that. Another option could be to use the IDAES generic property framework to define liquid and solid. Whichever path we take, I think this test can be reserved for a subsequent PR and noted in your master issue for UV, but if you think it'd be quick and easy for you to implement, I won't stop you!